demo/src/style/app.scss

```diff
@@ -101,10 +106,54 @@ p.github-buttons > iframe {
   background-color: transparent
 }
 
-div.component-api,
-div.component-api div.component-api-property,
-div.component-api div.component-api-instructions {
-  margin-bottom: 10px;
+div.api-doc-component {
+  margin-bottom: 20px;
+}
+
+div.api-doc-component > h2 > a {
+  text-decoration: none;
+}
+
+div.api-doc-component > h2 > a:hover {
+  text-decoration: none;
+}
+
+div.api-doc-component > h2 {
+  margin-top: 0.2em;
+  margin-bottom: 0.2em;
+  line-height:0.85em;
```

Comment:
Add space after `:`.

Also, do we want to be using `em`? It would be nice if we could use `rem`